discovery/kubernetes/kubernetes.go

```diff
@@ -130,13 +130,6 @@ func (c *NamespaceDiscovery) UnmarshalYAML(unmarshal func(interface{}) error) er
 func init() {
 	prometheus.MustRegister(eventCount)
 
-	// Initialize metric vectors.
-	for _, role := range []string{"endpoints", "node", "pod", "service", "ingress"} {
-		for _, evt := range []string{"add", "delete", "update"} {
-			eventCount.WithLabelValues(role, evt)
-		}
-	}
```

Comment:
This is a clean up for #6765. Now k8s sd use global variables so we don't need to init them here